shell/app/modules/project/types/issue.d.ts

```diff
@@ -237,3 +237,14 @@ declare namespace ISSUE {
   }
 }
 
+interface DrawerData {
+  [k: string]: any;
+  assignee: string;
+  bugStage: 'demandDesign';
```

Comment:
fixed value field? if value is fixed then you can push backend developer to ignore it